PlayCover/ViewModel/StoreVM.swift

```diff
@@ -20,23 +19,43 @@ class StoreVM: ObservableObject, @unchecked Sendable {
         resolveSources()
     }
 
+    @Published var enabledList: [String] = (UserDefaults.standard.stringArray(forKey: "enableSourceList") ??
+                                             StoreVM.shared.sourcesList.map { $0.source })
+
     @Published var sourcesList: [SourceData] {
         didSet {
             encode()
         }
     }
     @Published var sourcesData: [SourceJSON] = [] {
         didSet {
-            sourcesApps.removeAll()
-            for source in sourcesData {
-                appendSourceData(source)
-            }
+            updateSourcesApps()
         }
     }
     @Published var sourcesApps: [SourceAppsData] = []
 
     private var resolveTask: Task<Void, Never>?
 
+    //
```

Comment:
No empty comments.

Reply:
I added it 'cause there are other "//" before func in that file